### PR TITLE
Gamedb: add missing serials

### PR DESCRIPTION
**Pull request by Mrlinkwii**
### Description of Changes

adds missing serial to Gamedb 
### Rationale behind Changes

A more complete Gamedb 
### Suggested Testing Steps

look at CI

### CODE REVIEW

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -160,6 +169,9 @@ PBPX-95601:
 PCPX-96311:
   name: "Gran Turismo 3 Trial Disk Volume 1"
   region: "NTSC-J"
+PCPX-96321:
+  name: "sky Gunner [Trial]"
```
Comment (RedDevilus) [inline bin/resources/GameIndex.yaml:173]:
```suggestion
  name:SkyGunner [Trial]"
```

Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:173]: capital S please.

File: bin/resources/GameIndex.yaml [hunk 2]
```diff
@@ -21360,9 +21378,21 @@ SLPM-60207:
...
     eeClampMode: 3 # For camera issues in some scenes.
+SLPM-60239:
+  name: "Katamari Damacy [Trial]"
+  region: "NTSC-J"
+  roundModes:
+    vuRoundMode: 0 # Fixes SPS.
+  clampModes:
+    vuClampMode: 3 # Fixes SPS.
+  speedHacks:
+    mvuFlagSpeedHack: 0 # Fixes performance and falling through floor and other gameplay.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:21389]: Fixes performance, falling through the floor and other gameplay problems.

File: bin/resources/GameIndex.yaml [hunk 3]
```diff
@@ -21360,9 +21378,21 @@ SLPM-60207:
...
 SLPM-60243:
   name: "Full House Kiss [Trial]"
   region: "NTSC-J"
+SLPM-60246:
+  name: "Burnout 3: Takedown [Trial]"
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:21394]: Hyphen instead of a colon please.

File: bin/resources/GameIndex.yaml [hunk 4]
```diff
@@ -76,6 +79,9 @@ PAPX-90516:
     mipmap: 1
     textureInsideRT: 1
     autoFlush: 1
+PAPX-90517:
+  name: "Prince of Persia: Jikan no Suna [Trial]"
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:83]: Hyphen instead of colon please

File: bin/resources/GameIndex.yaml [hunk 5]
```diff
@@ -35,6 +35,9 @@ GUST-00009:
 PAPX-90201:
   name: "Fantavision [Taikenban]"
   region: "NTSC-J"
+PAPX-90202:
+  name: "IQ Remix: Intelligent Qube [Trial]"
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:39]: Hyphen instead of colon please.

File: bin/resources/GameIndex.yaml [hunk 6]
```diff
@@ -21360,9 +21378,21 @@ SLPM-60207:
...
+  clampModes:
+    vuClampMode: 3 # Fixes SPS.
+  speedHacks:
+    mvuFlagSpeedHack: 0 # Fixes performance and falling through floor and other gameplay.
 SLPM-60243:
   name: "Full House Kiss [Trial]"
   region: "NTSC-J"
+SLPM-60246:
+  name: "Burnout 3: Takedown [Trial]"
+  region: "NTSC-J"
```
Comment (RedDevilus) [inline bin/resources/GameIndex.yaml:21395]: clampModes:
    vuClampMode: 0  # Fixes buggy lighting on certain objects.
  gsHWFixes:
    halfPixelOffset: 2 # Fixes depth lines.
    
    Check the other gamefixes for other serials too.

File: bin/resources/GameIndex.yaml (hunk 4 above)
Comment (RedDevilus) [reply bin/resources/GameIndex.yaml:83]: gsHWFixes:
    autoFlush: 1
    
If you can read romaji it says Sands of time and this is what the other serials contain.

**Review by RedDevilus — COMMENTED** [review]
Rest of the serials looks fine.

File: bin/resources/GameIndex.yaml [hunk 7]
```diff
@@ -21349,6 +21366,14 @@ SLPM-60109:
 SLPM-60123:
   name: "Gekikuukan Pro Baseball: The End of the Century 1999 [Trial]"
   region: "NTSC-J"
+SLPM-60138:
+  name: "Klonoa 2: Lunatea's Veil [Trial]"
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:21370]: Hyphen please

**Pull request merged**